### PR TITLE
DOC: Clarify DataFrame.combine_first and Series.combine_first

### PR DESCRIPTION
**Pull request by HeidiNeuhaeuser**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -2989,12 +2989,15 @@ def combine(self, other, func, fill_value=None) -> Series:
 
     def combine_first(self, other) -> Series:
         """
-        Combine Series values, choosing the calling Series's values first.
+        Update null elements with value in the same location in 'other'.
+
+        Combine two Series objects by filling null values in one Series with
+        non-null values from the other Series.
```
Comment (MarcoGorelli) [inline pandas/core/series.py:2995]: per the corresponding line in `frame.py`, perhaps mention that the resulting index will be the union of the two?

File: pandas/core/series.py [hunk 2]
```diff
@@ -2989,12 +2989,15 @@ def combine(self, other, func, fill_value=None) -> Series:
...
 
         Parameters
         ----------
         other : Series
-            The value(s) to be combined with the `Series`.
+            The value(s) to be used for filling null values in `Series`.
```
Comment (MarcoGorelli) [inline pandas/core/series.py:3000]: perhaps just "The value(s) to be used for filling null values"?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @HeidiNeuhaeuser , I like the example

I just have some minor suggestions

Also, it would be good to share the docstrings between the Series and DataFrame methods (see `to_markdown`, among others, for an example of how this is one), either here or in a follow-up pull request

File: pandas/core/series.py (hunk 1 above)
Comment (HeidiNeuhaeuser) [reply pandas/core/series.py]: It was in the "Notes" section, I moved that for consistency.

**Review by MarcoGorelli — APPROVED** [review]
Looks good, thanks - do you want to try making a shared docstring (like there is in `to_markdown` and some other methods) or should I open that as a good first issue for someone else to take on?

**Comment by jreback** [comment]
thanks @HeidiNeuhaeuser if you'd like to make a shared doc-string as suggested by @MarcoGorelli pls open a new PR

**Pull request merged**